### DISCUSSION TITLE
Disable updates in system context?

### DISCUSSION THREAD

**Discussion by oskdanAX** [Weatherlights/Winget-AutoUpdate-Intune#28] [Q&A]
I'm utilizing the blocklist: 
![image](https://github.com/Weatherlights/Winget-AutoUpdate-Intune/assets/130646284/03f6521b-0f5f-42a7-a550-a9ad93472dca)
and it's synced down to C:\ProgramData\Winget-AutoUpdate-Configurator\excluded_apps.txt.
Although, at every reboot it tries to install those same applications but in system context.
![image](https://github.com/Weatherlights/Winget-AutoUpdate-Intune/assets/130646284/6bd07913-1c98-4d9f-bdf9-fb7e9b96aca9)
Which require the user to provide ADM account which they do not have. 

How do I disable this, I'd prefer the updates are only being upgraded as user context. 
I have this enabled as well: 
![image](https://github.com/Weatherlights/Winget-AutoUpdate-Intune/assets/130646284/cc785706-c570-4585-8329-c4ac087cb948)

**Comment by Weatherlights** [comment]
Currently the systemcontext execution can not be disabled. But a system context installation should not require administrative permission. I guess your problem is an app from the usercontext is updated but has a dependency to an app that requires the system account?
